### ISSUE TITLE
Schedule-maker error in standard workflow (No such file or directory)

### ISSUE THREAD

**Issue by daren-thomas** [architecture-building-systems/CityEnergyAnalyst#2440]
Steps to reproduce:

- new Virtual Machine
- install CityEnergyAnalyst 2.25
- start CityEnergyAnalyst-GUI
- create new project
- create new Scenario
  - tick all the tools (btw: the dialog for the weather file is weird)
- try to run `data-helper` (doesn't work because schedules are missing)
- try to run `schedule-maker` (output: see below)

```
C:\Users\user\Documents\CityEnergyAnalyst
λ cea schedule-maker
City Energy Analyst version 2.25
Running `cea schedule-maker` with the following parameters:
- general:scenario = C:\Users\user\Documents\projects\baseline
  (default: {general:project}\{general:scenario-name})
- general:multiprocessing = True
  (default: True)
- general:number-of-cpus-to-keep-free = 1
  (default: 1)
- general:debug = False
  (default: False)
- schedule-maker:buildings = []
  (default: [])
- schedule-maker:schedule-model = deterministic
  (default: deterministic)
Running occupancy model for scenario C:\Users\user\Documents\projects\baseline
Running occupancy model  with schedule model=deterministic
Using single process
Traceback (most recent call last):
  File "C:\Users\user\Documents\CityEnergyAnalyst\Dependencies\Python\lib\runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "C:\Users\user\Documents\CityEnergyAnalyst\Dependencies\Python\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "C:\Users\user\Documents\CityEnergyAnalyst\Dependencies\Python\Scripts\cea.exe\__main__.py", line 7, in <module>
  File "C:\Users\user\Documents\CityEnergyAnalyst\Dependencies\Python\lib\site-packages\cea\interfaces\cli\cli.py", line 64, in main
    script_module.main(config)
  File "C:\Users\user\Documents\CityEnergyAnalyst\Dependencies\Python\lib\site-packages\cea\demand\schedule_maker\schedule_maker.py", line 466, in main
    schedule_maker_main(locator, config)
  File "C:\Users\user\Documents\CityEnergyAnalyst\Dependencies\Python\lib\site-packages\cea\demand\schedule_maker\schedule_maker.py", line 85, in schedule_maker_main
    repeat(stochastic_schedule, n))
  File "C:\Users\user\Documents\CityEnergyAnalyst\Dependencies\Python\lib\site-packages\cea\utilities\parallel.py", line 140, in wrapper
    result = func(*instance_args)
  File "C:\Users\user\Documents\CityEnergyAnalyst\Dependencies\Python\lib\site-packages\cea\demand\schedule_maker\schedule_maker.py", line 128, in calc_schedules
    schedule = read_cea_schedule(locator.get_building_weekly_schedules(building))
  File "C:\Users\user\Documents\CityEnergyAnalyst\Dependencies\Python\lib\site-packages\cea\utilities\schedule_reader.py", line 44, in read_cea_schedule
    with open(path_to_cea_schedule) as f:
IOError: [Errno 2] No such file or directory: 'C:\\Users\\user\\Documents\\projects\\baseline\\inputs\\building-properties\\schedules\\B1000.csv'
```

**Comment by daren-thomas** [comment]
After checking with @martin-mosteiro, it seems the workflow is to run the `data-helper` with `schedules` selected.

**Comment by daren-thomas** [comment]
This is the error I get when running the `data-helper`:

```
λ cea data-helper
City Energy Analyst version 2.25
Running `cea data-helper` with the following parameters:
- general:scenario = C:\Users\user\Documents\projects\baseline
  (default: {general:project}\{general:scenario-name})
- data-helper:region = CH
  (default: CH)
- data-helper:overwrite-technology-folder = True
  (default: True)
- data-helper:databases = ['comfort', 'architecture', 'HVAC', 'internal-loads', 'supply', 'schedules']
  (default: ['comfort', 'architecture', 'HVAC', 'internal-loads', 'supply', 'schedules'])
- data-helper:buildings = []
  (default: [])
Running data-helper with scenario = C:\Users\user\Documents\projects\baseline
Running data-helper with archetypes = ['comfort', 'architecture', 'HVAC', 'internal-loads', 'supply', 'schedules']
Copying technology databases from C:\Users\user\Documents\CityEnergyAnalyst\Dependencies\Python\lib\site-packages\cea\databases\CH
C:\Users\user\Documents\CityEnergyAnalyst\Dependencies\Python\lib\site-packages\cea\datamanagement\data_helper.py:291: UnicodeWarning: Unicode equal comparison failed to convert both arguments to Unicode - interpreting them as being unequal
  (indexed_df.loc[building, use] == indexed_df.max(axis=1)[building]) and (use != 'PARKING')]
Traceback (most recent call last):
  File "C:\Users\user\Documents\CityEnergyAnalyst\Dependencies\Python\lib\runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "C:\Users\user\Documents\CityEnergyAnalyst\Dependencies\Python\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "C:\Users\user\Documents\CityEnergyAnalyst\Dependencies\Python\Scripts\cea.exe\__main__.py", line 7, in <module>
  File "C:\Users\user\Documents\CityEnergyAnalyst\Dependencies\Python\lib\site-packages\cea\interfaces\cli\cli.py", line 64, in main
    script_module.main(config)
  File "C:\Users\user\Documents\CityEnergyAnalyst\Dependencies\Python\lib\site-packages\cea\datamanagement\data_helper.py", line 512, in main
    buildings=buildings)
  File "C:\Users\user\Documents\CityEnergyAnalyst\Dependencies\Python\lib\site-packages\cea\datamanagement\data_helper.py", line 213, in data_helper
    calc_mixed_schedule(locator, building_occupancy_df, buildings)
  File "C:\Users\user\Documents\CityEnergyAnalyst\Dependencies\Python\lib\site-packages\cea\datamanagement\schedule_helper.py", line 45, in calc_mixed_schedule
    schedule_data_all_uses = ScheduleData(locator)
  File "C:\Users\user\Documents\CityEnergyAnalyst\Dependencies\Python\lib\site-packages\cea\datamanagement\schedule_helper.py", line 163, in __init__
    self.schedule_data, self.schedule_complementary_data = self.fill_in_data()
  File "C:\Users\user\Documents\CityEnergyAnalyst\Dependencies\Python\lib\site-packages\cea\datamanagement\schedule_helper.py", line 167, in fill_in_data
    for file in os.listdir(self.path_database):
WindowsError: [Error 3] The system cannot find the path specified: 'C:\\Users\\user\\Documents\\projects\\baseline\\inputs\\technology\\archetypes\\schedules\\*.*'
```

**Comment by daren-thomas** [comment]
OK. It turns out that the MANIFEST.in was not updated to include CSV schedules for the archetypes. Updating this for release-2.25.1 on PyPI.

(NOTE: this bug only manifests itself when not running in developer mode)

**Comment by daren-thomas** [comment]
ok. this is being fixed in the release-2.25 branch. closing.
